code/game/machinery/slotmachine.dm

```diff
@@ -227,23 +254,25 @@
 	var/linelength = get_lines()
 
 	if(reels[1][2] + reels[2][2] + reels[3][2] + reels[4][2] + reels[5][2] == "[SEVEN][SEVEN][SEVEN][SEVEN][SEVEN]")
-		visible_message("<b>[src]</b> says, 'JACKPOT! You win [money] credits worth of coins!'")
+		visible_message("<b>[src]</b> says, 'JACKPOT! You win [money] credits!'")
 		priority_announce("Congratulations to [user ? user.real_name : usrname] for winning the jackpot at the slot machine in [get_area(src)]!")
 		jackpots += 1
-		balance += money - give_coins(JACKPOT)
+		balance += money - give_payout(JACKPOT)
 		money = 0
-
-		for(var/i = 0, i < 5, i++)
-			var/cointype = pick(subtypesof(/obj/item/coin))
-			var/obj/item/coin/C = new cointype(loc)
-			random_step(C, 2, 50)
+		if(paymode == HOLOCHIP)
+			new /obj/item/holochip(loc,JACKPOT)
+		else
+			for(var/i = 0, i < 5, i++)
+				cointype = pick(subtypesof(/obj/item/coin))
+				var/obj/item/coin/C = new cointype(loc)
+				random_step(C, 2, 50)
```

Comment:
holochips give you a flat 10000 while coins can have varying rewards (most likely lower than 10K). Is that intended?

Reply:
The coin code got me confused, now I get it. It gives 10k coins (during give_payout, line 260) and 5 extra coins of any kind (Could be plasma, bananium, etc). What I did here was actually double the payout in case of holochips. Should I keep the coin as "extra prize" for jackpot or do you suggest something else (maybe a 1k extra holochip instead of 10k?)

Reply:
idc lol I didn't read the entire file as long as you know what's going on

Reply:
So, 20 grand if you get a jackpot with holocredits and 10 grand + 5 random coins if in coins?

Reply:
don't matter to me although long term I'd really want physical money and holochips to be interchangeable